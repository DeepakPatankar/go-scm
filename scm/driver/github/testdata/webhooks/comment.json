{
    "action": "created",
    "issue": {
      "url": "https://api.github.com/repos/wings-software/harness-ci-webhook-test/issues/2",
      "repository_url": "https://api.github.com/repos/wings-software/harness-ci-webhook-test",
      "labels_url": "https://api.github.com/repos/wings-software/harness-ci-webhook-test/issues/2/labels{/name}",
      "comments_url": "https://api.github.com/repos/wings-software/harness-ci-webhook-test/issues/2/comments",
      "events_url": "https://api.github.com/repos/wings-software/harness-ci-webhook-test/issues/2/events",
      "html_url": "https://github.com/wings-software/harness-ci-webhook-test/pull/2",
      "id": 735678167,
      "node_id": "MDExOlB1bGxSZXF1ZXN0NTE1MDI2Njgw",
      "number": 2,
      "title": "Update README.md",
      "user": {
        "login": "aman-harness",
        "id": 10278482,
        "node_id": "MDQ6VXNlcjEwMjc4NDgy",
        "avatar_url": "https://avatars0.githubusercontent.com/u/10278482?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/aman-harness",
        "html_url": "https://github.com/aman-harness",
        "followers_url": "https://api.github.com/users/aman-harness/followers",
        "following_url": "https://api.github.com/users/aman-harness/following{/other_user}",
        "gists_url": "https://api.github.com/users/aman-harness/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/aman-harness/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/aman-harness/subscriptions",
        "organizations_url": "https://api.github.com/users/aman-harness/orgs",
        "repos_url": "https://api.github.com/users/aman-harness/repos",
        "events_url": "https://api.github.com/users/aman-harness/events{/privacy}",
        "received_events_url": "https://api.github.com/users/aman-harness/received_events",
        "type": "User",
        "site_admin": false
      },
      "labels": [
  
      ],
      "state": "open",
      "locked": false,
      "assignee": null,
      "assignees": [
  
      ],
      "milestone": null,
      "comments": 4,
      "created_at": "2020-11-03T22:32:14Z",
      "updated_at": "2020-12-29T05:49:14Z",
      "closed_at": null,
      "author_association": "COLLABORATOR",
      "active_lock_reason": null,
      "pull_request": {
        "url": "https://api.github.com/repos/wings-software/harness-ci-webhook-test/pulls/2",
        "html_url": "https://github.com/wings-software/harness-ci-webhook-test/pull/2",
        "diff_url": "https://github.com/wings-software/harness-ci-webhook-test/pull/2.diff",
        "patch_url": "https://github.com/wings-software/harness-ci-webhook-test/pull/2.patch"
      },
      "body": "",
      "performed_via_github_app": null
    },
    "comment": {
      "url": "https://api.github.com/repos/wings-software/harness-ci-webhook-test/issues/comments/751955664",
      "html_url": "https://github.com/wings-software/harness-ci-webhook-test/pull/2#issuecomment-751955664",
      "issue_url": "https://api.github.com/repos/wings-software/harness-ci-webhook-test/issues/2",
      "id": 751955664,
      "node_id": "MDEyOklzc3VlQ29tbWVudDc1MTk1NTY2NA==",
      "user": {
        "login": "aman-harness",
        "id": 10278482,
        "node_id": "MDQ6VXNlcjEwMjc4NDgy",
        "avatar_url": "https://avatars0.githubusercontent.com/u/10278482?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/aman-harness",
        "html_url": "https://github.com/aman-harness",
        "followers_url": "https://api.github.com/users/aman-harness/followers",
        "following_url": "https://api.github.com/users/aman-harness/following{/other_user}",
        "gists_url": "https://api.github.com/users/aman-harness/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/aman-harness/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/aman-harness/subscriptions",
        "organizations_url": "https://api.github.com/users/aman-harness/orgs",
        "repos_url": "https://api.github.com/users/aman-harness/repos",
        "events_url": "https://api.github.com/users/aman-harness/events{/privacy}",
        "received_events_url": "https://api.github.com/users/aman-harness/received_events",
        "type": "User",
        "site_admin": false
      },
      "created_at": "2020-12-29T05:49:14Z",
      "updated_at": "2020-12-29T05:49:14Z",
      "author_association": "COLLABORATOR",
      "body": "trigger abc",
      "performed_via_github_app": null
    },
    "repository": {
      "id": 309651765,
      "node_id": "MDEwOlJlcG9zaXRvcnkzMDk2NTE3NjU=",
      "name": "harness-ci-webhook-test",
      "full_name": "wings-software/harness-ci-webhook-test",
      "private": true,
      "owner": {
        "login": "wings-software",
        "id": 18273000,
        "node_id": "MDEyOk9yZ2FuaXphdGlvbjE4MjczMDAw",
        "avatar_url": "https://avatars1.githubusercontent.com/u/18273000?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/wings-software",
        "html_url": "https://github.com/wings-software",
        "followers_url": "https://api.github.com/users/wings-software/followers",
        "following_url": "https://api.github.com/users/wings-software/following{/other_user}",
        "gists_url": "https://api.github.com/users/wings-software/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/wings-software/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/wings-software/subscriptions",
        "organizations_url": "https://api.github.com/users/wings-software/orgs",
        "repos_url": "https://api.github.com/users/wings-software/repos",
        "events_url": "https://api.github.com/users/wings-software/events{/privacy}",
        "received_events_url": "https://api.github.com/users/wings-software/received_events",
        "type": "Organization",
        "site_admin": false
      },
      "html_url": "https://github.com/wings-software/harness-ci-webhook-test",
      "description": "Webhook test",
      "fork": false,
      "url": "https://api.github.com/repos/wings-software/harness-ci-webhook-test",
      "forks_url": "https://api.github.com/repos/wings-software/harness-ci-webhook-test/forks",
      "keys_url": "https://api.github.com/repos/wings-software/harness-ci-webhook-test/keys{/key_id}",
      "collaborators_url": "https://api.github.com/repos/wings-software/harness-ci-webhook-test/collaborators{/collaborator}",
      "teams_url": "https://api.github.com/repos/wings-software/harness-ci-webhook-test/teams",
      "hooks_url": "https://api.github.com/repos/wings-software/harness-ci-webhook-test/hooks",
      "issue_events_url": "https://api.github.com/repos/wings-software/harness-ci-webhook-test/issues/events{/number}",
      "events_url": "https://api.github.com/repos/wings-software/harness-ci-webhook-test/events",
      "assignees_url": "https://api.github.com/repos/wings-software/harness-ci-webhook-test/assignees{/user}",
      "branches_url": "https://api.github.com/repos/wings-software/harness-ci-webhook-test/branches{/branch}",
      "tags_url": "https://api.github.com/repos/wings-software/harness-ci-webhook-test/tags",
      "blobs_url": "https://api.github.com/repos/wings-software/harness-ci-webhook-test/git/blobs{/sha}",
      "git_tags_url": "https://api.github.com/repos/wings-software/harness-ci-webhook-test/git/tags{/sha}",
      "git_refs_url": "https://api.github.com/repos/wings-software/harness-ci-webhook-test/git/refs{/sha}",
      "trees_url": "https://api.github.com/repos/wings-software/harness-ci-webhook-test/git/trees{/sha}",
      "statuses_url": "https://api.github.com/repos/wings-software/harness-ci-webhook-test/statuses/{sha}",
      "languages_url": "https://api.github.com/repos/wings-software/harness-ci-webhook-test/languages",
      "stargazers_url": "https://api.github.com/repos/wings-software/harness-ci-webhook-test/stargazers",
      "contributors_url": "https://api.github.com/repos/wings-software/harness-ci-webhook-test/contributors",
      "subscribers_url": "https://api.github.com/repos/wings-software/harness-ci-webhook-test/subscribers",
      "subscription_url": "https://api.github.com/repos/wings-software/harness-ci-webhook-test/subscription",
      "commits_url": "https://api.github.com/repos/wings-software/harness-ci-webhook-test/commits{/sha}",
      "git_commits_url": "https://api.github.com/repos/wings-software/harness-ci-webhook-test/git/commits{/sha}",
      "comments_url": "https://api.github.com/repos/wings-software/harness-ci-webhook-test/comments{/number}",
      "issue_comment_url": "https://api.github.com/repos/wings-software/harness-ci-webhook-test/issues/comments{/number}",
      "contents_url": "https://api.github.com/repos/wings-software/harness-ci-webhook-test/contents/{+path}",
      "compare_url": "https://api.github.com/repos/wings-software/harness-ci-webhook-test/compare/{base}...{head}",
      "merges_url": "https://api.github.com/repos/wings-software/harness-ci-webhook-test/merges",
      "archive_url": "https://api.github.com/repos/wings-software/harness-ci-webhook-test/{archive_format}{/ref}",
      "downloads_url": "https://api.github.com/repos/wings-software/harness-ci-webhook-test/downloads",
      "issues_url": "https://api.github.com/repos/wings-software/harness-ci-webhook-test/issues{/number}",
      "pulls_url": "https://api.github.com/repos/wings-software/harness-ci-webhook-test/pulls{/number}",
      "milestones_url": "https://api.github.com/repos/wings-software/harness-ci-webhook-test/milestones{/number}",
      "notifications_url": "https://api.github.com/repos/wings-software/harness-ci-webhook-test/notifications{?since,all,participating}",
      "labels_url": "https://api.github.com/repos/wings-software/harness-ci-webhook-test/labels{/name}",
      "releases_url": "https://api.github.com/repos/wings-software/harness-ci-webhook-test/releases{/id}",
      "deployments_url": "https://api.github.com/repos/wings-software/harness-ci-webhook-test/deployments",
      "created_at": "2020-11-03T10:36:21Z",
      "updated_at": "2020-12-29T05:06:28Z",
      "pushed_at": "2020-12-29T05:06:25Z",
      "git_url": "git://github.com/wings-software/harness-ci-webhook-test.git",
      "ssh_url": "git@github.com:wings-software/harness-ci-webhook-test.git",
      "clone_url": "https://github.com/wings-software/harness-ci-webhook-test.git",
      "svn_url": "https://github.com/wings-software/harness-ci-webhook-test",
      "homepage": null,
      "size": 338096,
      "stargazers_count": 0,
      "watchers_count": 0,
      "language": "Java",
      "has_issues": true,
      "has_projects": true,
      "has_downloads": true,
      "has_wiki": true,
      "has_pages": false,
      "forks_count": 0,
      "mirror_url": null,
      "archived": false,
      "disabled": false,
      "open_issues_count": 2,
      "license": null,
      "forks": 0,
      "open_issues": 2,
      "watchers": 0,
      "default_branch": "master"
    },
    "organization": {
      "login": "wings-software",
      "id": 18273000,
      "node_id": "MDEyOk9yZ2FuaXphdGlvbjE4MjczMDAw",
      "url": "https://api.github.com/orgs/wings-software",
      "repos_url": "https://api.github.com/orgs/wings-software/repos",
      "events_url": "https://api.github.com/orgs/wings-software/events",
      "hooks_url": "https://api.github.com/orgs/wings-software/hooks",
      "issues_url": "https://api.github.com/orgs/wings-software/issues",
      "members_url": "https://api.github.com/orgs/wings-software/members{/member}",
      "public_members_url": "https://api.github.com/orgs/wings-software/public_members{/member}",
      "avatar_url": "https://avatars1.githubusercontent.com/u/18273000?v=4",
      "description": ""
    },
    "sender": {
      "login": "aman-harness",
      "id": 10278482,
      "node_id": "MDQ6VXNlcjEwMjc4NDgy",
      "avatar_url": "https://avatars0.githubusercontent.com/u/10278482?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/aman-harness",
      "html_url": "https://github.com/aman-harness",
      "followers_url": "https://api.github.com/users/aman-harness/followers",
      "following_url": "https://api.github.com/users/aman-harness/following{/other_user}",
      "gists_url": "https://api.github.com/users/aman-harness/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/aman-harness/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/aman-harness/subscriptions",
      "organizations_url": "https://api.github.com/users/aman-harness/orgs",
      "repos_url": "https://api.github.com/users/aman-harness/repos",
      "events_url": "https://api.github.com/users/aman-harness/events{/privacy}",
      "received_events_url": "https://api.github.com/users/aman-harness/received_events",
      "type": "User",
      "site_admin": false
    }
  }