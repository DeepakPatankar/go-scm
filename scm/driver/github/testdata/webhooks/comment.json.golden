{
	"Action": "",
	"Repo": {
		"ID": "309651765",
		"Namespace": "wings-software",
		"Name": "harness-ci-webhook-test",
		"Perm": null,
		"Branch": "master",
		"Private": true,
		"Visibility": 0,
		"Clone": "https://github.com/wings-software/harness-ci-webhook-test.git",
		"CloneSSH": "git@github.com:wings-software/harness-ci-webhook-test.git",
		"Link": "https://github.com/wings-software/harness-ci-webhook-test",
		"Created": "0001-01-01T00:00:00Z",
		"Updated": "0001-01-01T00:00:00Z"
	},
	"Issue": {
		"Number": 2,
		"Title": "Update README.md",
		"Body": "",
		"Link": "https://github.com/wings-software/harness-ci-webhook-test/pull/2",
		"Labels": null,
		"Closed": false,
		"Locked": false,
		"Author": {
			"Login": "aman-harness",
			"Name": "",
			"Email": "",
			"Avatar": "https://avatars0.githubusercontent.com/u/10278482?v=4",
			"Created": "0001-01-01T00:00:00Z",
			"Updated": "0001-01-01T00:00:00Z"
		},
		"Created": "2020-11-03T22:32:14Z",
		"Updated": "2020-12-29T05:49:14Z"
	},
	"Comment": {
		"ID": 751955664,
		"Body": "trigger abc",
		"Author": {
			"Login": "aman-harness",
			"Name": "",
			"Email": "",
			"Avatar": "https://avatars0.githubusercontent.com/u/10278482?v=4",
			"Created": "0001-01-01T00:00:00Z",
			"Updated": "0001-01-01T00:00:00Z"
		},
		"Created": "2020-12-29T05:49:14Z",
		"Updated": "2020-12-29T05:49:14Z"
	},
	"Sender": {
		"Login": "aman-harness",
		"Name": "",
		"Email": "",
		"Avatar": "https://avatars0.githubusercontent.com/u/10278482?v=4",
		"Created": "0001-01-01T00:00:00Z",
		"Updated": "0001-01-01T00:00:00Z"
	}
}